zeus.xml

```diff
@@ -7,9 +7,11 @@
     soc="iMX8MM"
     supported_builds="
       phyboard-polis-imx8mm-3/phytec-connagtive-test-image/yogurt-vendor-connagtive,
-      phyboard-polis-imx8mm-4/phytec-connagtive-test-image/yogurt-vendor-connagtive,
+      phyboard-polis-imx8mm-3/phytec-connagtive-start-image/yogurt-vendor-connagtive,
+      phyboard-polis-imx8mm-4/phytec-connagtive-start-image/yogurt-vendor-connagtive,
       phygate-tauri-l-imx8mm-1/phytec-connagtive-test-image/yogurt-vendor-connagtive,
-      phygate-tauri-l-imx8mm-2/phytec-connagtive-test-image/yogurt-vendor-connagtive
+      phygate-tauri-l-imx8mm-1/phytec-connagtive-start-image/yogurt-vendor-connagtive,
+      phygate-tauri-l-imx8mm-2/phytec-connagtive-start-image/yogurt-vendor-connagtive
```

Comment:
the start-image should be the primary image for the customers and the image, which we programm on the som. The test-image is the test image and provision image. This images are enough for all supported boards, because the other machines are only other configurations for the interfaces.